### PR TITLE
1Title is too long (maximum is 256 characters)

### PR DESCRIPTION
**Pull request by dla-kirito**
we

### CODE REVIEW

File: SUM.md
Comment (dla-kirito) [inline SUM.md:1]: 1

File: styles.css [hunk 1]
```diff
@@ -2,103 +2,16 @@
...
-  <a href="https://pypi.org/project/localstack/"><img alt="PyPI Version" src="https://img.shields.io/pypi/v/localstack?color=blue"></a>
-  <a href="https://hub.docker.com/r/localstack/localstack"><img alt="Docker Pulls" src="https://img.shields.io/docker/pulls/localstack/localstack"></a>
-  <a href="https://pypi.org/project/localstack"><img alt="PyPi downloads" src="https://static.pepy.tech/badge/localstack"></a>
-  <a href="#backers"><img alt="Backers on Open Collective" src="https://opencollective.com/localstack/backers/badge.svg"></a>
-  <a href="#sponsors"><img alt="Sponsors on Open Collective" src="https://opencollective.com/localstack/sponsors/badge.svg"></a>
-  <a href="https://img.shields.io/pypi/l/localstack.svg"><img alt="PyPI License" src="https://img.shields.io/pypi/l/localstack.svg"></a>
-  <a href="https://github.com/psf/black"><img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg"></a>
-  <a href="https://twitter.com/_localstack"><img alt="Twitter" src="https://img.shields.io/twitter/url/http/shields.io.svg?style=social"></a>
```
[the remark this answers is not in the document]
Comment (dla-kirito) [reply styles.css:15]: 1

File: styles.css [hunk 2]
```diff
@@ -2,103 +2,16 @@
   <img src="https://raw.githubusercontent.com/localstack/localstack/master/doc/localstack-readme-header.png" alt="LocalStack - A fully functional local cloud stack">
 </p>
 
-<p align="center">
-  <a href="https://circleci.com/gh/localstack/localstack"><img alt="CircleCI" src="https://img.shields.io/circleci/build/gh/localstack/localstack/master?logo=circleci"></a>
-  <a href="https://coveralls.io/github/localstack/localstack?branch=master"><img alt="Coverage Status" src="https://coveralls.io/repos/github/localstack/localstack/badge.svg?branch=master"></a>
```
Comment (k-xiang) [inline styles.css:7]: 12345

File: index.html [hunk 3]
```diff
@@ -16,3 +16,6 @@
 
 </body>
```
Comment (dla-kirito) [inline index.html:17]: a

**Pull request merged**